src/pages/home/report/PureReportActionItem.tsx

```diff
@@ -0,0 +1,1222 @@
+                isDisabled={draftMessage !== undefined}
+                shouldFreezeCapture={isPaymentMethodPopoverActive}
+            >
+                {(hovered) => (
+                    <View style={highlightedBackgroundColorIfNeeded}>
+                        {shouldDisplayNewMarker && (!shouldUseThreadDividerLine || !isFirstVisibleReportAction) && <UnreadActionIndicator reportActionID={action.reportActionID} />}
+                        {shouldDisplayContextMenu && (
+                            <MiniReportActionContextMenu
+                                reportID={reportID}
+                                reportActionID={action.reportActionID}
+                                anchor={popoverAnchorRef}
+                                originalReportID={originalReportID}
+                                isArchivedRoom={isArchivedRoom}
+                                displayAsGroup={displayAsGroup}
+                                disabledActions={disabledActions}
+                                isVisible={hovered && draftMessage === undefined && !hasErrors}
+                                isThreadReportParentAction={isThreadReportParentAction}
+                                draftMessage={draftMessage}
+                                isChronosReport={isChronosReport}
+                                checkIfContextMenuActive={toggleContextMenuFromActiveReportAction}
+                                setIsEmojiPickerActive={setIsEmojiPickerActive}
+                            />
+                        )}
+                        <View
+                            style={StyleUtils.getReportActionItemStyle(
+                                hovered || isWhisper || isContextMenuActive || !!isEmojiPickerActive || draftMessage !== undefined || isPaymentMethodPopoverActive,
+                                draftMessage === undefined && !!onPress,
+                            )}
+                        >
+                            <OfflineWithFeedback
+                                onClose={() => {
+                                    const transactionID = ReportActionsUtils.isMoneyRequestAction(action) ? ReportActionsUtils.getOriginalMessage(action)?.IOUTransactionID : undefined;
+                                    if (transactionID) {
+                                        clearError(transactionID);
+                                    }
+                                    clearAllRelatedReportActionErrors(reportID, action);
+                                }}
+                                // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+                                pendingAction={
+                                    draftMessage !== undefined ? undefined : action.pendingAction ?? (action.isOptimisticAction ? CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD : undefined)
+                                }
+                                shouldHideOnDelete={!isThreadReportParentAction}
```

Comment:
We should've used `isDeletedParentAction` for check here. We fixed it [here](https://github.com/Expensify/App/issues/62587)